java/BinExport/extension.properties

```diff
@@ -1,5 +1,5 @@
-name=BinExport
+name=@extname@
```

Comment:
I'd prefer it if we keep this as is, it's unlikely to change, esp. not between Ghidra versions.